src/librustc_typeck/check/method/probe.rs

```diff
@@ -799,7 +807,38 @@ impl<'a, 'gcx, 'tcx> ProbeContext<'a, 'gcx, 'tcx> {
             return Err(MethodError::PrivateMatch(def, out_of_scope_traits));
         }
 
+        debug!("Probing for method names similar to {:?}",
```

Comment:
It might be better to extract this section to a separate method?

Reply:
Done!